dspace-api/pom.xml

```diff
@@ -717,7 +827,7 @@
 
         <dependency>
             <groupId>org.xmlunit</groupId>
-            <artifactId>xmlunit-matchers</artifactId>
+            <artifactId>xmlunit-core</artifactId>
```

Comment:
Same here...why the switch to -core?  This seems new as well as it's not on `main` branch.

Reply:
All of the classes that we actually use are in -core, not -matchers.  `mvn dependency:analyze` pointed this out.  I don't know why we were using -matchers.

Reply:
Ah, thanks for clarification! This seems correct then